### ISSUE TITLE
Add Contributing section

### ISSUE THREAD

**Issue by cvan** [WebVRRocks/webvrrocks#35]

**Comment by cvan** [comment]
A decent stub: https://github.com/WebVRRocks/webvrrocks/blob/master/OUTLINE.md#contributing

**Comment by cvan** [comment]
Some layout/content inspiration:

* https://community.algolia.com/ (a bit visual and overwhelming)
* https://www.rust-lang.org/en-US/community.html (a bit verbose but content is easily searchable within the page)
* https://www.consul.io/community.html
* https://www.packer.io/community/
* https://www.terraform.io/community.html
* https://teamweek.com/about.html

**Comment by caseyyee** [comment]
I like how the rust community page is super clear and straightforward.   Even if it is a bit wordy.   

The other idea in the examples that are good is how contributors are highlighted on the page.
